dissect/btrfs/stream.py

```diff
@@ -6,13 +6,9 @@
 from typing import TYPE_CHECKING, BinaryIO, NamedTuple
 from uuid import UUID
 
+from dissect.util import lzo
```

Comment:
```suggestion
from dissect.util.compression import lzo
```
I think I like this a bit better.